.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.2.2
+current_version = 3.2.3
```

Comment:
can you remove the version changes here and in __init ? We will do it with the cli